.lando.yml

```diff
@@ -68,3 +75,8 @@ tooling:
     description: Run tests by filter
     cmd:
       - appserver: composer run filter_test
+  db-import-custom:
```

Comment:
Add new command for performing custom database import. Can remove after https://github.com/lando/core/pull/384 is released